### PR TITLE
TSV report of the enums and permissible values #38 

### PR DESCRIPTION
**Pull request by turbomam**
#38

`linkml_model_enrichment/utils/enums_pvs_tsv.py`

### CODE REVIEW

File: linkml_model_enrichment/utils/enums_pvs_tsv.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+logger = logging.getLogger(__name__)
+click_log.basic_config(logger)
+
+
+@click.command()
+@click_log.simple_verbosity_option(logger)
+@click.option('--modelfile', help="path to LinkML input", type=click.Path(exists=True), show_default=True,
+              required=True)
+@click.option('--output_file', default="enums_pvs_tsv.tsv", help="path to TSV output", type=click.Path(),
```
Comment (cmungall) [inline linkml_model_enrichment/utils/enums_pvs_tsv.py:15]: not good practice to include a default that relies on hardcoded files

File: linkml_model_enrichment/utils/enums_pvs_tsv.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+              show_default=True)
```
Comment (cmungall) [inline linkml_model_enrichment/utils/enums_pvs_tsv.py:16]: add a help=... for all options

File: linkml_model_enrichment/utils/enums_pvs_tsv.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+def enums_pvs_tsv(modelfile, output_file):
```
Comment (cmungall) [inline linkml_model_enrichment/utils/enums_pvs_tsv.py:17]: add a docstring

**Comment by turbomam** [comment]
Thanks, applying your requests now

Sorry about the terse issue and PR descriptions. This generates a TSV report of the enums and permissible values in a LinkML YAML file.

File: linkml_model_enrichment/utils/enums_pvs_tsv.py (hunk 2 above)
Comment (turbomam) [reply linkml_model_enrichment/utils/enums_pvs_tsv.py:16]: which option doesn't have help?

File: linkml_model_enrichment/utils/enums_pvs_tsv.py (hunk 1 above)
Comment (turbomam) [reply linkml_model_enrichment/utils/enums_pvs_tsv.py:15]: shoot, that's been a habit of mine. not a good practice even though it's just a filename, with no path?

I changed it anyway.

**Review by cmungall — APPROVED** [review]

**Pull request merged**